examples/reference/panes/Perspective.ipynb

```diff
@@ -17,82 +21,250 @@
+    "    'datetime': [(datetime.now() + timedelta(hours=i)) for i in range(9)],\n",
+    "    'category': ['Category A', 'Category B', 'Category C', 'Category A', 'Category B',\n",
+    "             'Category C', 'Category A', 'Category B', 'Category C',],\n",
+    "    'link': ['https://panel.holoviz.org/', 'https://discourse.holoviz.org/', 'https://github.com/holoviz/panel']*3,\n",
+    "}\n",
+    "df = pd.DataFrame(data)\n",
     "\n",
-    "pn.pane.Perspective(data, width=1000)"
+    "pn.pane.Perspective(df, width=1000)"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "Try interacting with the `Perspective` pane.\n",
+    "\n",
+    "- The three vertical dots in the upper, left corner will toggle the *configuration menu*\n",
+    "- The three vertical lines at the top of each column will toggle a *column configuration menu*.\n",
+    "- The top menu will provide you with options to change the *plugin* as well as *group*, *split*, *order* and *filter* your data\n",
+    "- The bottom menu will provide you with options to *reset*, *download* and *copy* as well as *change the theme*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
    "source": [
-    "You can also provide configuration options for the viewer plugins:"
+    "The `index` is shown by default. If you want to not display it by default, you can provide the list of `columns` to display."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "outputs": [],
    "source": [
-    "random_df = pd.DataFrame(np.random.randn(20, 2), columns=list('XY'))\n",
+    "pn.pane.Perspective(df, columns=list(df.columns), width=1000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "You might also hide the *config menu* via the `toggle_config` parameter."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "pn.pane.Perspective(df, columns=[\"float\"], group_by=[\"category\"], plugin='d3_y_bar', toggle_config=False, width=1000, height=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Try interacting with the `Perspective` pane by toggling the *config menu* via the 3 vertical dots in the upper, left corner."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Plugin Configuration\n",
     "\n",
-    "pn.pane.Perspective(random_df, width=1000, plugin_config={\n",
+    "You can configure the active *plugin* manually as shown for the *Datagrid* below\n",
+    "\n",
+    "![Manual DataGrid Plugin Column Configuration](https://user-images.githubusercontent.com/42288570/244842791-79a94dcf-2d4a-4b8c-ace7-3c0904682a31.png)"
```

Comment:
Please move this image to the cdn @philippjfr and update the link.

Reply:
We'll give you access to 1password and by proxy the AWS bucket at some point.